### ISSUE TITLE
As an administrator, I want to have more fine-grained analytics access settings.

### ISSUE THREAD

**Issue by Tapchicoma** [pullreminders/backlog#168]
As an administrator, I want an option to disable leaderboard for the organization, as it brings in unnecessary competition and pressure on the developers.

We want Pullreminder be considered inside a company as a help tool that improves PR turnaround, but not as a tool that company may use for evaluating developer productivity.

**Comment by abinoda** [comment]
@Tapchicoma There's a way to restrict access – https://docs.pullpanda.com/products/pull-analytics/user-access

Go to `/installs/:accountId/settings/analytics_permissions` to access, replace `accountId` with yours.

**Comment by Tapchicoma** [comment]
But this will also close access to analytics, that we want it to be open to the team :thinking: 

Leaderboard analytics is not directly accessible in the analytics tab. I would rephrase then this ticket to following:
"As an administrator, I want to have more fine-grained analytics access settings."

**Comment by abinoda** [comment]
@Tapchicoma Got it. Concern over the leaderboards comes up from time to time...

What we recommend (and has worked for other customers who've initially had similar concerns) is that you (or management) explicitly communicate to your team that the leaderboard metrics are meant to be a fun way to create more awareness, and that they are NOT to be used for any sort of performance evaluation or assessment purposes. Once this is made clear to the team we've seen that the sensitivity goes away.

What do you think?

**Comment by abinoda** [comment]
And of course, the name "leaderboard" doesn't help. We're aware of this and considering renaming it to something else.

**Comment by Tapchicoma** [comment]
> What we recommend (and has worked for other customers who've initially had similar concerns) is that you (or management) explicitly communicate to your team that the leaderboard metrics are meant to be a fun way to create more awareness, and that they are NOT to be used for any sort of performance evaluation or assessment purposes. Once this is made clear to the team we've seen that the sensitivity goes away.

We've already done that, but people still may want to compete with each other. Renaming "leaderboard" would be nice, but disabling it completely would be better :slightly_smiling_face:

**Comment by abinoda** [comment]
@Tapchicoma Gotcha. Will keep this issue open – definitely want to revisit this. Also curious to know if the sensitivity subsides for your team as some time passes.

**Comment by petemounce** [comment]
Would it be possible to provide a setting that we could fill in some text to such that that text (along the lines of "these are vanity metrics, not intended for decision-making") appear prominently on the leaderboard (and maybe other) pages?
